frontend/src/pages/NewStudylogPage/index.tsx

```diff
@@ -25,20 +32,52 @@ type SelectOption = { value: string; label: string };
 
 const NewStudylogPage = () => {
   const history = useHistory();
-
   const editorContentRef = useRef<any>(null);
+  const { user } = useContext(UserContext);
+  const { username } = user;
 
   useBeforeunload(editorContentRef);
 
   const [studylogContent, setStudylogContent] = useState<NewStudylogForm>({
     title: '',
-    content: '',
+    content: null,
     missionId: null,
     sessionId: null,
     tags: [],
     abilities: [],
   });
 
+  const fetchTempSavedStudylogRequest: UseQueryResult<
+    AxiosResponse<TempSavedStudyLog>,
+    AxiosError
+  > = useQuery(
+    [REACT_QUERY_KEY.TEMP_STUDYLOG, username],
+    () =>
+      requestGetTempSavedStudylog({
+        accessToken: localStorage.getItem(LOCAL_STORAGE_KEY.ACCESS_TOKEN) as string,
+      }),
+    {
+      refetchOnWindowFocus: false,
+      onSuccess: ({ data }) => {
+        const isTempSavedStudylogExist = Object.keys(data).some((key) => data[key] !== null);
```

Comment:
오홍.. 임시저장된게 있는지 확인하는 로직이군요! 
임시저장된게 없으면 모두 null 값으로 오나보군요.
그럼, Response 타이핑에 `Nullable` 이 적용되어야 할것같아요.

Exist가 없어도 유무를 파악하기 적절한 네이밍인 것 같아요! 
++ `Object.entries`를 쓰면 좀 더 직관적인 것같습니다!

```suggestion
        const isTempSavedStudylog = Object.entries(data).some(([_, value]) => value !== null);
```

Reply:
isTempSavedStudylog라고 한다면, 임시저장된 스터디 로그인지?라고 해석이 될 것 같아서 존재하는 지를 나타내기 위해 Exist를 붙였었습니다! 없어도 이해하기 편했던 건가요?!

Reply:
널러블 타입 적용했습니다

[1d08acb](https://github.com/woowacourse/prolog/pull/1067/commits/1d08acb827be633a87e4c419a4974d78c05a2ac3)

Reply:
말씀하신대로 해석의 혼동이 있을 수 있을거같아요.
지금도 좋아요~

Reply:
ㅎㅎ좋습니다!